scripts/HFIR_4Circle_Reduction/absorption.py

```diff
@@ -0,0 +1,363 @@
+import numpy
+import numpy.linalg
+import math
+from mantid.api import AnalysisDataService
+
+# Do absorption correction
+
+
+class Lattice(object):
```

Comment:
Just tried to run the same script as the scientist.  It will be deleted later in my code.